snuba/querylog/query_metadata.py

```diff
@@ -254,7 +254,7 @@ def to_dict(self) -> snuba_queries_v1.Querylog:
         # TODO: Remove check once Org IDs are required
         if org_id := self.request.attribution_info.tenant_ids.get("organization_id"):
             if isinstance(org_id, int):
-                request_dict["organization"] = org_id
+                request_dict["organization"] = max(org_id, 0)
```

Comment:
if we aren't sending to the DLQ, maybe we should emit a metric or a sentry warning?

Reply:
If you did want to put it in DLQ instead of writing to clickhouse and treating it as valid, you can just raise an exception and it should automatically happen

Reply:
we figured out that this unfortunately is done on the produce side, so it's not a good place to raise that exception